integreat_cms/core/management/commands/find_missing_versions.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from argparse import ArgumentTypeError
+
+from ....cms.models import PageTranslation, EventTranslation, POITranslation
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+#: The possible model choices
+MODELS = {"page": PageTranslation, "event": EventTranslation, "poi": POITranslation}
+
+
+def get_model(model_str):
+    """
+    Convert a model string to a translation model class
+
+    :param model_str: The model string
+    :type model_str: str
+
+    :return: The model class
+    :rtype: type
+
+    :raises ~argparse.ArgumentTypeError: When the input is invalid
+    """
+    try:
+        return MODELS[model_str]
+    except KeyError as e:
+        raise ArgumentTypeError(
+            "Invalid model (must be either page, event or poi)"
+        ) from e
+
+
+class Command(LogCommand):
+    """
+    Management command to missing versions
```

Comment:
I'm sorry if I'm a bit picky here, but I think there's still a verb missing here :sweat_smile: I think you mean "find"?

Reply:
Haha thanks, yes that's the right amount of picky-ness :laughing: